assimilation_code/programs/obs_sequence_tool/obs_sequence_tool.html

```diff
@@ -790,6 +803,35 @@ <h3>Subset by Observation or QC Value</h3>
    max_copy           = 300.0,
 </pre>
 
+<A NAME="precomputed"></A>
+<h3>Precomputed Forward Observation Values</h3>
+<P>
+Precomputed Forward Observation Values are the result of an external program
+that computes the expected observation values from an ensemble of model states
+and includes these values as part of the observation metadata (see, for example,
+the <em class=code>GSI2DART</em> observation converter).
+By default, any observation with precomputed forward observation (FO) values 
+will have those values simply pass through <em class=code>obs_sequence_tool</em>
+just like any other piece of metadata.  If the precomputed forward observation 
+values for any or all observation types are not wanted, it is possible to 
+remove the precomputed values and retain the rest of the observation. 
+<strong>Important</strong>: observations resulting from 
+<em class=code>perfect_model_obs</em> are <strong>not</strong> 
+precomputed forward observation values!
+</P>
+<pre>
+# keep all precomputed values from all observations with precomputed values (the default):
+   remove_precomputed_FO_values = ''
+
+# remove all precomputed values from all observations with precomputed values:
+   remove_precomputed_FO_values = 'ALL'
+
+# remove all precomputed values for specific observation types (case does not matter):
+# The observations themselves will still be present in the output, given no other
+# subsetting processing.
+   remove_precomputed_FO_values = 'RADIOSONDE_temperATURE', 'AMDAR_U_WIND_COMPONENT' 
```

Comment:
```suggestion
   remove_precomputed_FO_values = 'RADIOSONDE_TEMPERATURE', 'AMDAR_U_WIND_COMPONENT' 
```

Reply:
The bizarre case was intended to demonstrate that case does not matter. I agree it looks funny.